backend/rest/rest.go

```diff
@@ -20,6 +22,7 @@ type client struct {
 }
 
 var storage persistence.Storage
+var cachedRepos []*backends.Result
```

Comment:
am I missing something or are you just populating this without actually using it?

Reply:
leftover from a previous attempt, remove it